packages/orbit-components/src/Collapse/Collapse.stories.tsx

```diff
@@ -85,25 +113,20 @@ export const OpenedByDefault = ({ label }) => {
         maxValue={24}
       />
     </Collapse>
-  );
-};
-
-OpenedByDefault.story = {
-  name: "Opened by default",
+  ),
 
   parameters: {
     info: "You can try all possible configurations of this component. However, check Orbit.Kiwi for more detailed design guidelines.",
+    controls: {
+      exclude: ["initialExpanded", "expanded"],
+    },
   },
 };
 
-OpenedByDefault.args = {
-  label: "Duration",
-};
-
-export const WithActions = ({ label }) => {
-  return (
+export const WithActions: Story = {
```

Comment:
There is `<No Display Name` in the source instead of real component name, can you fix it, please?